src/Games/NexusMods.Games.AdvancedInstaller.UI/AdvancedManualInstallerUI.cs

```diff
@@ -70,7 +71,7 @@ public AdvancedManualInstallerUI(IServiceProvider provider)
 
     private async Task<(bool shouldInstall, DeploymentData data)> GetDeploymentDataAsync(
         GameInstallation gameInstallation, string modName,
-        FileTreeNode<RelativePath, ModSourceFileEntry> archiveFiles)
+        KeyedBox<RelativePath, ModFileTree> archiveFiles)
```

Comment:
I just had a quick look through, 
since we use `KeyedBox<RelativePath, ModFileTree>` a lot, can we make a type alias for it that is shorter and either has Node or Tree in the name? 

I think something simple like `RelativeModFileTree` works. To improve readability and shorten it.

Reply:
I agree with this, I had to dig through the path library to find out what KeyedBox was, it turns out it's a wrapper for `TVal` with an additional `TKey` *typed* tag, the Key isn't passed along with the box, just the type of the Key. We might want to find a better name for these in the mainline parts of the application.

Reply:
I'm open to suggestions personally.
Heck, even `ModFileTreeNode` works for me.

Reply:
Go ahead with `ModFileTreeNode`. We can change it later if we come up with a better name.